lib/moonshot.rb

```diff
@@ -10,8 +10,12 @@ module BuildMechanism # rubocop:disable Documentation
   end
   module DeploymentMechanism # rubocop:disable Documentation
   end
+  module Plugins # rubocop:disable Documentation
+  end
 end
 
+require_relative 'plugins/backup'
```

Comment:
I don't think we should concern ourselves with that yet. There are many files that not every user uses.

If we want to autoload because of slow startup then we should address that all at once.